enterprise/frontend/test/metabase-enterprise/sandboxes/sandboxes.cy.spec.js

```diff
@@ -772,5 +775,131 @@ describeWithToken("formatting > sandboxes", () => {
         },
       );
     });
+
+    it.skip("advanced sandboxing should not ignore data model features like object detail of FK (metabase-enterprise#520)", () => {
+      const COLLECTION_GROUP_ID = 4;
+
+      withSampleDataset(({ PRODUCTS_ID, ORDERS_ID }) => {
+        cy.log("**-- 1. Create the first native question with a filter --**");
+
+        cy.request("POST", "/api/card", {
+          name: "EE_520_Q1",
+          dataset_query: {
+            database: 1,
+            native: {
+              query:
+                "SELECT * FROM ORDERS WHERE USER_ID={{sandbox}} AND TOTAL>10",
+              "template-tags": {
+                sandbox: {
+                  "display-name": "Sandbox",
+                  id: "1115dc4f-6b9d-812e-7f72-b87ab885c88a",
+                  name: "sandbox",
+                  type: "text",
```

Comment:
Just to avoid any problems in the future in case something gets more strict.
```suggestion
                  type: "number",
```

Reply:
Excellent thinking!
I will take this into account when writing future tests as well.